tests/test_signature.py

```diff
@@ -114,6 +114,35 @@ def test_expand_varargs():
     assert s.expand_varargs(4) == (int, int, float, float)
 
 
+def test_varargs_tie_breaking():
+    assert Sig(int) < Sig(int, varargs=int)
+    assert not Sig(int) >= Sig(int, varargs=int)
+    assert Sig(int, varargs=int) < Sig(int, Num)
+    assert not Sig(int, varargs=int) >= Sig(int, Num)
+    assert Sig(int, int, varargs=int) < Sig(int, Num)
+    assert not Sig(int, int, varargs=int) >= Sig(int, Num)
+
+
```

Comment:
In #119 I also had added those tests. You might want to include them as well...

```suggestion

    assert not (Sig(int, Num) < Sig(int, varargs=int))
    assert not (Sig(int, varargs=Num) < Sig(int, varargs=int))
    assert not (Sig(int, varargs=Num) <= Sig(int, varargs=int))

    class A:
        pass

    class B:
        pass

    assert not Sig(int, varargs=A) < Sig(int, varargs=B)
    assert not Sig(int, varargs=B) < Sig(int, varargs=A)

    assert not Sig(int, varargs=Num) < Sig(Num, varargs=int)
    assert not Sig(Num, varargs=int) < Sig(int, varargs=Num)


```